pkg/app/master/commands/clifvgetter.go

```diff
@@ -239,12 +240,12 @@ func GetContainerOverrides(ctx *cli.Context) (*config.ContainerOverrides, error)
 		return nil, envErr
 	}
 	env := ctx.StringSlice(FlagEnv)
-	envList = append(envList, env...)
+	envList = validateAndCleanEnvVariables(append(envList, env...), xc)
```

Comment:
Here you are taking on responsibility for validating the env vars specified with the `--env` flag and the env vars loaded from a file. If this is your intent then you'll need to update your logic and the logic in `ParseEnvFile` because it already does basic env var format validation. Another option is to exclude the env var validation loaded from a file.

Reply:
I see. I figured that was the case, but could not find a way to test out putting in key/values through ParseEnvFile. I'll take the last option then on this one. Thanks!